packages/caravan-psbt/package.json

```diff
@@ -54,18 +54,14 @@
     "@caravan/eslint-config": "*",
     "@caravan/multisig": "*",
     "@caravan/typescript-config": "*",
-    "@inrupt/jest-jsdom-polyfills": "^3.2.1",
-    "@jest/globals": "^29.7.0",
-    "@types/jest": "^29.5.12",
+    "@vitejs/plugin-react": "^4.3.1",
```

Comment:
why was this necessary?

Reply:
I had moved that dependency to the root package.json file as of which I've removed it from here